src/Symfony/Component/Form/Form.php

```diff
@@ -910,7 +910,7 @@ public function add($child, $type = null, array $options = array())
             $options['auto_initialize'] = false;
 
             if (null === $type && null === $this->config->getDataClass()) {
-                $type = 'text';
+                $type = 'Symfony\Component\Form\Extension\Core\Type\TextType';
```

Comment:
this makes me think we are not covering this in the testsuite, as it would have triggered a deprecation warning